src/dune

```diff
@@ -1,4 +1,5 @@
 (library
  (name claudius)
  (public_name claudius)
- (libraries tsdl))
+ (libraries tsdl)
+ (modules base font framebuffer palette primitives screen utils run_loop))
```

Comment:
I don't believe there is any need to add this part of the library stanza. As per the dune docs: https://dune.readthedocs.io/en/stable/reference/dune/library.html

> Specifies what modules are part of the library. By default, Dune will use all the .ml/.re files in the same directory as the dune file.

It doesn't appear to me that this is needed here? 

As an aside, perhaps this change was part of the path to getting your PR. Do take care when _reviewing_ your own PR before submitting and ask yourself do we need all of these changes.